superduperdb/container/model.py

```diff
@@ -429,6 +431,22 @@ def __post_init__(self):
         else:
             self.to_call = getattr(self.object.artifact, self.predict_method)
 
+    @override
+    def on_load(self, db: DB) -> None:
+        if not hasattr(self.object.artifact, "to_method"):
+            return
+
+        for device in self.preferred_devices:
+            try:
+                getattr(self.object.artifact, "to_method")(device)
+                return
+            except Exception as e:
+                logging.warning(e)
+                pass
```

Comment:
you don't need pass

Reply:
@blythed do we need this? logging.warning

wouldn't it be a spam for all torch models on_load

Reply:
addressed.